pkg/api/alerting/v2alpha1/types.go

```diff
@@ -26,21 +26,27 @@ import (
 	"github.com/emicklei/go-restful"
 	"github.com/pkg/errors"
 	prommodel "github.com/prometheus/common/model"
+	"github.com/prometheus/prometheus/pkg/timestamp"
 	"github.com/prometheus/prometheus/promql/parser"
+	"github.com/prometheus/prometheus/template"
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 )
 
 const (
 	RuleLevelCluster   RuleLevel = "cluster"
 	RuleLevelNamespace RuleLevel = "namespace"
+
+	AnnotationKeyRuleUpdateTime = "rule_update_time"
```

Comment:
normally, annotation key format follows `rule-update-time`, using slash instead of underscode

Reply:
This annotation key is intended for use within prometheus, so following its common usage

Reply:
here is an example of an annotation used in the alerts Prometheus sent:
https://github.com/kubesphere/notification-manager/blob/master/pkg/webhook/testdata/alert.json#L17